api/java/bingo-nosql/src/test/java/com/epam/indigo/BingoTests.java

```diff
@@ -23,4 +23,28 @@ void checkBingoVersion(@TempDir Path tempDir) {
         );
         bingo.close();
     }
+
+    @Test
+    @DisplayName("Testing enumerateId from Java bindings")
+    void checkEnumerateId(@TempDir Path tempDir) {
+        Indigo indigo = new Indigo();
+        Bingo bingo = Bingo.createDatabaseFile(indigo, tempDir.toString(), "molecule");
+        bingo.insert(indigo.loadMolecule("C"));
+        bingo.insert(indigo.loadMolecule("CC"));
+        BingoObject enumerator = bingo.enumerateId();
+        int i = 0;
+        while (enumerator.next()) {
+            assertEquals(
+                i,
+                enumerator.getCurrentId(),
```

Comment:
The same here:
Why we may rely upon that `Id` of the object in the enumerator is equal to the sequential number?
This is nowhere specified.